pyanaconda/ui/gui/spokes/custom.py

```diff
@@ -489,9 +467,42 @@ def get_new_devices(self):
 
         return new_devices
 
+    def _set_page_label_text(self):
+        if self._accordion.is_multiselection:
+            select_tmpl = _("{fmt_start}{selected}{fmt_end} of "
```

Comment:
This is going to be a mess to translate. There are a lot pieces that will mean nothing to translators, which creates the potential for the translations to break it, and I'm not really sure what the purpose of all the fmt_start and fmt_end is anyway. The entire string is going to be made black or gray, and not just the pieces wrapped in span tags, right? I can see that fmt_start+fmt_end will also make it large and bold, but I don't understand what the color adds at that point. Either way, I think it would be more readable for both us and for translators if all of, for example, "{fmt_start}{selected}{fmt_end}" was a single parameter in the format string.

Additionally, it would be nicer if this used the old-style % formatting that we use elsewhere in anaconda. For one, I'm old and I hate new things, and for two, the {} style is not regular which makes some of the analysis we do in the tests a giant pain.